src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioBaseDiagnosticListTable.cs

```diff
@@ -482,60 +479,6 @@ private string GetHelpLink(DiagnosticData item)
                         return null;
                     }
 
-                    public bool TryCreateColumnContent(int index, string columnName, bool singleColumnView, out FrameworkElement content)
-                    {
-                        content = default(FrameworkElement);
-                        if (columnName != ShimTableColumnDefinitions.ErrorCode)
-                        {
-                            return false;
-                        }
-
-                        var item = GetItem(index);
-                        if (item == null)
-                        {
-                            return false;
-                        }
-
-                        Uri unused;
-                        if (BrowserHelper.TryGetUri(item.HelpLink, out unused))
-                        {
-                            content = GetOrCreateTextBlock(ref _errorCodes, this.Count, index, item, i => GetHyperLinkTextBlock(i, new Uri(i.HelpLink, UriKind.Absolute), bingLink: false));
```

Comment:
Does this mean we no longer construct the bing hyperlink based on 'MessageFormat'? How does error list construct the hyperlink?

We still construct hyperlink based on MessageFormat in preview pane - do we need to change that?